src/main/java/org/broadinstitute/hellbender/utils/hdf5/HDF5PoN.java

```diff
@@ -35,60 +35,65 @@
  */
 public final class HDF5PoN implements PoN {
 
-    public final static String TARGET_FACTORS_GROUP_NAME = "/target_factors";
+    private final HDF5File file;
 
-    private final static String NORMALIZED_PCOV_GROUP_NAME = "/fnt_control_matrix";
+    /*===============================================================================================================*
+     * FIELDS FOR COVERAGE DATA                                                                                      *
+     *===============================================================================================================*/
 
-    private final static String VERSION_GROUP_NAME = "/version";
+    public static final String TARGET_FACTORS_GROUP_NAME = "/target_factors";
```

Comment:
Was there are reason for making these public? Also I believe you moved this file in a different location in a later commit